### PR TITLE
DBZ-4999 fixing current_timestamp for Informix 12.10

### PR DESCRIPTION
**Pull request by nrkljo**
* fixing current_timestamp for Informix 12.10
* tuning ifx cdc parameters
* workflows

### CODE REVIEW

**Comment by jpechane** [comment]
Hi, I am sorry for the delay, the PR got lost in my inbox.
As Informix connector was already released, could you please create a separate Jira describing the issues and attribute the commits to it?
Also I see CI failing, could you please try to rebase to the latest `main`?

File: pom.xml [hunk 1]
```diff
@@ -287,7 +285,7 @@
                                 </wait>
                             </run>
                             <build>
-                                <from>${informix.image}:12.10.FC12W1DE</from>
+                                <from>ibmcom/informix-developer-database:12.10.FC12W1DE</from>
```
Comment (jpechane) [inline pom.xml:288]: Why is the image name hardcoded?

File: pom.xml [hunk 2]
```diff
@@ -330,7 +328,7 @@
                                 </wait>
                             </run>
                             <build>
-                                <from>${informix.image}:14.10.FC9W1DE</from>
+                                <from>icr.io/informix/informix-developer-database:14.10.FC9W1DE</from>
```
Comment (jpechane) [inline pom.xml:331]: There is a different registry used here. Is it intentional? If yes then maybe a new Maven property should be introduced that will hold the registry name. And the `from` will be constructed frem registry name property and image name property

File: src/main/java/io/debezium/connector/informix/InformixEventMetadataProvider.java [hunk 3]
```diff
@@ -20,39 +20,29 @@ public class InformixEventMetadataProvider implements EventMetadataProvider {
...
+        final Long timestamp = value.getStruct(Envelope.FieldName.SOURCE).getInt64(SourceInfo.TIMESTAMP_KEY);
         return timestamp == null ? null : Instant.ofEpochMilli(timestamp);
     }
 
     @Override
     public Map<String, String> getEventSourcePosition(DataCollectionId source, OffsetContext offset, Object key, Struct value) {
-        if (value == null) {
-            return null;
+        if (value == null || source == null) {
+            return Map.of();
```
Comment (jpechane) [inline src/main/java/io/debezium/connector/informix/InformixEventMetadataProvider.java:33]: Why is this different from other connectors that return `null`?

**Comment by nrkljo** [comment]
Hi Jiri,

OK, yes I will. I wasn’t sure I could create Jiras, the “create” button didn’t appear for me before; I see it does now.
I’ll take your comments on-board and withdraw this one, and resubmit under a new Jira.

Med vänlig hälsning
Lars M. Johansson
Konsult, IT-Arkitekt

File: pom.xml (hunk 2 above)
Comment (nrkljo) [reply pom.xml:331]: Yes, the latest Informix 14.10 image is in the IBM Container Registry under the 'informix' label while the 12.10 images are with the older 14.10 images in Docker Hub, under the 'ibmcom' label. 
I'll sew something up...

File: src/main/java/io/debezium/connector/informix/InformixEventMetadataProvider.java (hunk 3 above)
Comment (nrkljo) [reply src/main/java/io/debezium/connector/informix/InformixEventMetadataProvider.java:33]: Sorry, "magic refactoring" by IDEA, need to turn that off...

**Comment by nrkljo** [comment]
Withdrawing and resubmitting under [DBZ-7137](https://issues.redhat.com/browse/DBZ-7137)

**Pull request closed without merging**